evm/src/main/java/org/hyperledger/besu/evm/frame/MessageFrame.java

```diff
@@ -1349,6 +1349,7 @@ public static class Builder {
     private Multimap<Address, Bytes32> accessListWarmStorage = HashMultimap.create();
 
     private Optional<List<Hash>> versionedHashes;
+
```

Comment:
we can remove this modification

Reply:
it is voluntary, to add a missing space to separate 2 methods